sdk/python/tests/compiler/compiler_tests.py

```diff
@@ -141,6 +142,15 @@ def test_basic_workflow(self):
       shutil.rmtree(tmpdir)
       # print(tmpdir)
 
+  def test_convert_k8s_obj_to_dic(self):
+    converted = compiler.Compiler()._convert_k8s_obj_to_dic({
+      "ENV": "test",
+      "number": 3,
+      "list": [1,2,3],
+      "time": datetime.datetime.now()
+    })
+    assert isinstance(converted["time"], str)
```

Comment:
Can we verify the whole converted results? Something like:

expected = {
      "ENV": "test",
      "number": 3,
      "list": [1,2,3],
      "time": 'YouNeedToUseAFixedTime'
    }
self.assertEqual(expected, converted)